cip-drep-metadata/README.md

```diff
@@ -0,0 +1,241 @@
+
+#### `picture`
+- Optional 
+- A base 64 encoded profile picture
+- Moderation of this image must be handled on the client side to comply with their TOS
+- This SHOULD be treated as the profile picture of the individual
+
+#### `objectives`
+- Optional
+- A freeform text field with a maximum of 1000 characters
+- A short description of what the person believes nd what they want to achieve as a DRep
+
+#### `motivation`
+- Optional
+- A freeform text field with a maximum of 1000 characters
+- A short description of why they want to be a DRep, what personal and professional experiences have they had that have driven them to 
+
+#### `qualifications`
+- Optional
+- A freeform text field with a maximum of 1000 characters 
+- A space to list the qualifications that the subject of this metadata has that are relevant to being a DRep
+
+#### `references`
+- This CIP extends the `references` property from [CIP-100](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0100#high-level-description)
+- `references` contain the following sub-properties `type`, `label`, and `uri`
+- This CIP adds two `@type` identifiers "Identity" and "Link"
+- The creator of the metadata MUST add a `label`, this `label` SHOULD describe the source of the url, e.g. if it is a link to the DRep's X account then the `label` should say "X". If it is a personal website the `label` should say "Personal Website" rather than domain_name.com.
+- The `label` of each link SHOULD NOT be left blank, but if it is then tooling displaying links SHOULD still allow users to access the URL.
+- Each link MUST have exactly one `uri` (as specified in CIP-100) which MUST not be blank. If it is blank then tooling providers displaying the Link SHOULD NOT display any of the details of this link.
+- `uri` MUST be the full address to the souce of the intended data and MUST NOT be a shortened url i.e. one that has been created using a service such as bitly. This is to ensure that tooling can easily scan for malicious links and users of tooling who are reading the `uri` value can understand where the link goes.
+- Tools that have a user interface that allows users to navigate to the location in the `uri` of a link, SHOULD inform their user of the full url that they will be visiting and SHOULD also warn their user that they will be visiting a different website and ask them whether they want to proceed.
+- Tooling providers SHOULD warn people that none of the information contained in external is verified by the tool and they should DYOR
```

Comment:
@Thomas-Upfield @Crypto2099 this is related to the still-unresolved question in https://github.com/cardano-foundation/CIPs/pull/788#discussion_r1533301310.

Reply:
I have removed this, but will add a note to tooling providers as a reminder to them rather than a prescription about whether they MUST or should do something.